src/api/projects/queries/getProject/index.js

```diff
@@ -23,6 +26,41 @@ const resolver = async (
     .eager('[locales, translationKeys.translationValues]')
     .findOne({ slug: projectSlug, ownerId: request.user.id })
 
+  const translationKeysCount = project.translationKeys.length
```

Comment:
☝️ since we’re gonna paginate them, we’ll have to switch to a count for that value